src/main/resources/com/google/jenkins/plugins/computeengine/ui/helpers/ProvisioningType/help-maxRunDurationSeconds.html

```diff
@@ -0,0 +1,19 @@
+<!--
+Copyright 2024 CloudBees, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div>
+    The maximum duration (in seconds) after which the VM will be automatically deleted by GCP.
+    See <a href="https://cloud.google.com/compute/docs/instances/limit-vm-runtime">Limit the run time of a VM</a> for more details.
```

Comment:
FYI, links in help files for config forms automatically open in a new tab/window as per some magic in Jenkins core—there is no need to manually right-click and select **Open in new tab** or similar.